community/mame/APKBUILD

```diff
@@ -48,11 +48,10 @@ subpackages="
 _groovymame_patch=groovymame-$pkgver.diff
 source="
 	https://github.com/mamedev/mame/archive/$pkgname$_pkgver.tar.gz
-	$_groovymame_patch::https://drive.google.com/uc?export=download&id=1_h5lcMQ3xMJlKcrh_u1CtCnQhrQgry7W
+	$_groovymame_patch::https://drive.google.com/uc?export=download&id=1K_JmYATSEwcqQx1JpD4ah42Txlg0H7bc
```

Comment:
Why is this patch not committed to aports.gir instead of downloaded?

Reply:
it is actually a distribution file which gets updated so often.
and is also too big.